Source/Particles/PulsarParameters.H

```diff
@@ -0,0 +1,30 @@
+#ifndef PULSAR_PARAMETERS_H
+#define PULSAR_PARAMETERS_H
+
+#include <AMReX_RealVect.H>
+#include <AMReX_REAL.H>
+#include <AMReX_GpuQualifiers.H>
+
+namespace PulsarParm
+{
+   extern std::string pulsar_type;
+
+   extern AMREX_GPU_DEVICE_MANAGED amrex::Real omega_star;
+   extern AMREX_GPU_DEVICE_MANAGED amrex::Real B_star;
+   extern AMREX_GPU_DEVICE_MANAGED amrex::Real R_star;
+   extern AMREX_GPU_DEVICE_MANAGED amrex::Real dR_star;
+   extern AMREX_GPU_DEVICE_MANAGED int EB_external;
+   extern AMREX_GPU_DEVICE_MANAGED int E_external_monopole;
+   extern AMREX_GPU_DEVICE_MANAGED amrex::Vector<amrex::Real> center_star;
```

Comment:
Declaring an `amrex::Vector` as managed will probably not work on the GPU since this class inherits from `std::vector` and doesn't provide any device functions.

It should work to declare `center_star` like:

```
extern AMREX_GPU_DEVICE_MANAGED amrex::GpuArray<amrex::Real, AMREX_SPACEDIM> center_star;
```

Reply:
Ah I see! Thank you Don